### ISSUE TITLE
Request for Training Dataset

### ISSUE THREAD

**Issue by tymons** [ankandrew/fast-plate-ocr#69]
Hello there,

First I wanted to say that I really appreciate your work and contribution to the edge licence plate detection. 

As I am trying to replicate and slightly modify your architecture in order to be deployed on MCUs was wondering if can you share training dataset for the OCR? Or maybe suggest some resources which you've used. Any tip will be appreciated!

Thanks,
Tymo

**Comment by ankandrew** [comment]
Hi, I'm glad you find my work useful :).

Although it's not in my current plans to release the full dataset, you can contribute a different arch you are mentioning and I can train it w/ full dataset and make it available in the model HUB.

If you need a quick dataset to validate the new model architecture, I'd merge/use [colombia_dataset_example.zip](https://github.com/ankandrew/fast-plate-ocr/releases/download/arg-plates/colombia_dataset_example.zip) + [arg_plate_dataset_plus_synth.zip](https://github.com/ankandrew/fast-plate-ocr/releases/download/arg-plates/arg_plate_dataset_plus_synth.zip), to get a simple dataset from these 2 countries. (They can be found in https://github.com/ankandrew/fast-plate-ocr/releases/tag/arg-plates).

**Comment by wanjunling168** [comment]
As far as I know, `ria-com/nomeroff-net` provides some big russian license plate dataset at their website for free: `https://nomeroff.net.ua/datasets/`, you can have a look at it and I found they also provided EU dataset at there

**Comment by wanjunling168** [comment]
Can you train a model with larger max_length (maybe 15 I think?), since I've found those 2-line plates (there even have 3-lines in Thailand) or Indian plates have more than 10 characters and hard to use your model to recognize @ankandrew

**Comment by ankandrew** [comment]
@wanjunling168 Yes, there shouldn't be a problem with very long plates or even 3-line ones. The reason the pre-trained models will fail most probably with Indian is 1. Because I didn't train with Indian plates 2. The maximum length (w/o padding) I used for the pre-trained model is 9 plate length.
